app/template/rest.py

```diff
@@ -55,9 +56,17 @@ def create_template(service_id):
 def update_template(service_id, template_id):
     fetched_template = dao_get_template_by_id_and_service_id(template_id=template_id, service_id=service_id)
 
+    data = request.get_json()
+
+    # if redacting, don't update anything else
+    if data.get('redact_personalisation') is True and 'updated_by_id' in data:
```

Comment:
added a test - nothing happens. Do not update, do not pass go, do not collect $200